core/controllers/story_viewer.py

```diff
@@ -141,7 +141,27 @@ def _record_node_completion(
         completed_node_ids: List[str],
         ordered_nodes: List[story_domain.StoryNode]
     ) -> Tuple[List[str], Optional[str], List[str]]:
-        """Records node completion."""
+        """Records node completion.
+
+        Args:
+            story_id: str. The story ID.
+            node_id: str. The node ID.
+            completed_node_ids: List[str]. A list of IDS of completed nodes.
+            ordered_nodes: List[story_domain.StoryNode]. A list of story
+                nodes in order.
+
+        Returns:
+            3-Tuple(next_exp_ids, next_node_id, completed_node_ids). Where:
+            next_exp_ids: List[str]. A list of exploration IDs
```

Comment:
I think you can indent this and the following lines.

What does "next" mean conceptually in "next_exp_ids"? How does this differ or relate to next_node_id?